sdk/daml-script/runner/src/test/resources/TestScript.daml

```diff
@@ -5,18 +5,18 @@ module TestScript where
 
 import Daml.Script
 
--- Each script gets the time to ensure the ledger is called
+-- Each script lists the known parties to ensure the ledger is called
 myScript : Script ()
 myScript = do
-  _ <- getTime
```

Comment:
these tests would have caught the missing protocol errors if it hadn't been for the fact that the default time mode is `--wall-clock-time`, which does _not_ call to the ledger :(